samples/bluetooth/mesh/light_ctrl/src/main.c

```diff
@@ -56,7 +59,11 @@ static void isr_emds_cb(void *arg)
 
 #if defined(CONFIG_BT_CTLR)
 	/* Stop mpsl to reduce power usage. */
```

Comment:
I guess this comment is not relevant anymore since this is mandatory in general, not for power save.